src/hal/qxspi/Qxi.c

```diff
@@ -2,7 +2,7 @@
 
 #include "Gpio.h"
 #include "Spi.h"
-#include "qxi.h"
+#include "include/Qxi.h"
```

Comment:
`#include include/<name>.h` is only used inside the library to include the header that is available when including this library to other code. 
If you use this library as an external dependency it will be included by `#include <name>.h`
This is done so the Header with library internal function prototypes isn't communicated to the outside.

Reply:
I've removed the `include` part from the files, because it is not necessary to have it. And for the unit-test dummies it would be a good idea to don't have them.